samples/msal-node-samples/ElectronSystemBrowserTestApp/.beachballrc

```diff
@@ -1,3 +0,0 @@
-{
-    "shouldPublish": false
```

Comment:
This file prevents our changelog generator from requiring changelogs for sample changes. Please restore.